Poker.AT/Poker.AT/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="NUnit" version="3.6.1" targetFramework="net45" />
+  <package id="NUnitLite" version="3.6.1" targetFramework="net452" />
```

Comment:
I need to see if the meaning of this is that we add nuget package to the project. but I think for now its ok